### ISSUE TITLE
bug(linux): Crash in ibus-keyman on first-time use after installation

### ISSUE THREAD

**Issue by ermshiperete** [keymanapp/keyman#10478]
I installed Keyman 17 alpha on Ubuntu 23.10 Mantic (X11). Don't remember if I rebooted or not. Opened keyman-config and added a keyboard and got a crash in ibus-keyman. The crash file (`/var/crash/_usr_libexec_ibus-engine-keyman.1000.crash`) indicated that the schema wasn't installed (which should have happened in `keyman.postinst`).

**Comment by ermshiperete** [comment]
Steps to reproduce on a fresh system:

- add keyman-alpha ppa: `sudo add-apt-repository ppa:keymanapp/keyman-alpha`
- install keyman: `sudo apt install keyman` 

It didn't crash if I upgraded ibus at the same time: `sudo apt install keyman ibus`. It didn't crash if I first upgraded ibus.

**Comment by ermshiperete** [comment]
The reason is that compiling the glib schemas happens after our postinst script runs.

```
Setting up ibus-keyman (17.0.256-1~sil1~mantic) ...
Setting up python3-bs4 (4.12.2-2) ...
Setting up python3-numpy (1:1.24.2-1) ...
Setting up python3-keyman-config (17.0.256-1~sil1~mantic) ...
Setting up keyman (17.0.256-1~sil1~mantic) ...
postinst called to compile glib schemas
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.36.0-1.1ubuntu1) ...
```

**Comment by ermshiperete** [comment]
See also #8996

**Comment by ermshiperete** [comment]
Closed via #10788
